compute/index.markdown

```diff
@@ -109,14 +109,26 @@ You can run all the same ssh commands and do what you need to, then once again s
 
 ## Google Compute Engine
 
-Google has [Compute Engine](https://cloud.google.com/products/compute-engine). To get your authorization key, visit the [Google API Console](https://code.google.com/apis/console/). Once there, go to "API Access". Click "Create another client ID..." and select "service account".
+Google has [Compute Engine](https://cloud.google.com/compute/). To get your authorization key, visit the [Google Developers Console](https://console.developers.google.com). Once there, select your project, go to "APIs & auth - Credentials". Click "Create another client ID..." and select "service account".
+
```

Comment:
Consider adding something like: Next, download either the JSON key (preferred) or the P12 key.

And then switch the order below so the JSON instructions come before P12.